app/models/spree/page.rb

```diff
@@ -39,20 +34,9 @@ def meta_title
 
 private
 
-  def update_positions_and_slug
+  def check_slug
```

Comment:
I wonder if this method should be called `check_slug`, considering that it actually alters the slug? Wouldn't `normalize_slug` be a better name?